### PR TITLE
TO-DO LIST PROJECT (USING WEBPACK)

### PR DESCRIPTION
**Pull request by direakanbi**
# TO-DO LIST PROJECT (WITH WEBPACK)
1. Initialized WebPack bundler and all the necessary plugins.
2. Added linters.
3. Rendered lists dynamically to the main page with OOP in JavaScript.
4. Added styling in relation to the given template.

### CODE REVIEW

**Comment by omar-labana** [comment]
Hi @direakanbi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have an updated README file about the project. Kindly, add a descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template).

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,39 @@
...
+
+# Built With
+  1. HTML
+  2. CSS
+  3. Linters (Webhint, Stylelint, Lighthouse).
+  4. Webpack Module Bundler.
+
+# Getting Started
+
+To get a local copy up and running follow these simple example steps.
```
Comment (Mihndim2020) [inline README.md]: - [ ]  As it a requirement for this project, which helpful for none technical users, some of which can be recruiters, please kindly outline the steps to;
  - clone your project locally e.g. `git clone https://github.com/direakanbi/To-Do-Project.git`,
  - install dependencies e.g. `nnm install`,
  - run the project locally. e.g. `npm start`

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @direakanbi, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Good job with PR summary
:heavy_check_mark: Good job passing linter checks

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by omar-labana** [comment]
Hi @direakanbi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 👏
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**